stage/Game.tsx

```diff
@@ -10,47 +10,42 @@ const Game = () => {
   const [opponentBoard, setOpponentBoard] = useState<number[][]>([]);
   const [playerBoard, setPlayerBoard] = useState<number[][]>([]);
 
-  const testShip: Ship = {
-    //enum = length
-    type: ShipNames["Destroyer"],
-  };
+  const Ships: Ship[] = [
```

Comment:
I think there may be a better way to call for this?
The ShipNames["xx"] returns an enum.

Reply:
You want it to return as enum, don't you? It seems that all it's for is to be assigned to the numbers?

Reply:
Probably better to explain on a call.